package.json

```diff
@@ -222,6 +222,7 @@
   },
   "resolutions": {
     "csstype": "3.0.11",
+    "meow": "8.1.2",
```

Comment:
yarn v4 has a much stricter resolution check on PRs and the lockfile seemed to have a v6 of meow listed which caused install to break. Resolving the whole monorepo to the codemod version fixed it